FluentTerminal.Models/KeyBinding.cs

```diff
@@ -1,14 +1,28 @@
-﻿using FluentTerminal.Models.Enums;
+﻿using System.Collections.Generic;
 
 namespace FluentTerminal.Models
 {
     public class KeyBinding
     {
-        public Command Command { get; set; }
         public int Key { get; set; }
         public bool Ctrl { get; set; }
         public bool Alt { get; set; }
         public bool Shift { get; set; }
         public bool Meta { get; set; }
+
+        public Dictionary<string, object> ToDict(AbstractCommand command)
```

Comment:
this seems odd, why does the keybinding know about commands? I think it would be easier to just change the Command property to string, then we can check if this string is either a guid of a profile or a string presentation of one of the command enums values.

Reply:
Problem is that the command was previously embedded in the KeyBinding object (https://github.com/felixse/FluentTerminal/blob/master/FluentTerminal.Models/KeyBinding.cs), but isn't anymore. It used to just serialize the keybindings list, which included the commands. Since the keybindings don't include commands anymore, we need something that adds it back in.

Reply:
we could add `public string Command { get; set; }` which is either Command.ToString() or shellProfile.Id.ToString()